### ISSUE TITLE
Incorrect error message with srcset attr image density descriptor value

### ISSUE THREAD

**Issue by kyohei-23** [markuplint/markuplint#1171]
- Markuplint version: `3.13.0` (VSCode extension) and Playground
- Parser lang: HTML
- Node.js version: `18.16.0`
- OS: WSL2 Ubuntu 22.04 on Windows11

## Describe the bug

Pixel density descriptor value is said below by HTML standard

> A pixel density descriptor, consisting of: [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace), a [valid floating-point number](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-floating-point-number) giving a number greater than zero representing the pixel density descriptor value, and a U+0078 LATIN SMALL LETTER X character.

> There must not be an [image candidate string](https://html.spec.whatwg.org/multipage/images.html#image-candidate-string) for an element that has the same [pixel density descriptor value](https://html.spec.whatwg.org/multipage/images.html#pixel-density-descriptor-value) as another [image candidate string](https://html.spec.whatwg.org/multipage/images.html#image-candidate-string)'s [pixel density descriptor value](https://html.spec.whatwg.org/multipage/images.html#pixel-density-descriptor-value) for the same element. For the purpose of this requirement, an [image candidate string](https://html.spec.whatwg.org/multipage/images.html#image-candidate-string) with no descriptors is equivalent to an [image candidate string](https://html.spec.whatwg.org/multipage/images.html#image-candidate-string) with a 1x descriptor.

Cite: https://html.spec.whatwg.org/multipage/images.html#srcset-attributes

But, the tool showes error when using `1x` value  as descriptor  in srcset attribute.

## Code Example

```html
<div>
  <img src="/path/to/file" alt="" srcset="/path/to/file 1x, /path/to/file@2x 2x" >
</div>
```

## Steps To Reproduce

1. Using default pixel density descriptor value `1x` in srcset attribute

## The current behavior
![image](https://github.com/markuplint/markuplint/assets/70838736/05bd9731-1713-45e1-972d-2071185b690a)

## The expected behavior
show no error when using 1x descriptor in `srcset` attribute
